changelog/21.0/21.0.0/summary.md

```diff
@@ -4,9 +4,9 @@
 
 - **[Major Changes](#major-changes)**
     - **[Deprecations and Deletions](#deprecations-and-deletions)**
+        - [Deprecated VTTablet Flags](#vttablet-flags)
         - [Deletion of deprecated metrics](#metric-deletion)
-        - [VTTablet Flags](#vttablet-flags)
-        - [Metrics](#deprecations-metrics)
+        - [Deprecated Metrics](#deprecations-metrics)
```

Comment:
It seems like we are using case, between `Deletion of deprecated metrics` and the other items (i.e. `Deprecated VTTablet Flags`)

Reply:
Good point, but I don't think it's very important.